proto/perp/v2/event.proto

```diff
@@ -50,6 +50,20 @@ message PositionChangedEvent {
 
   // The block number at which this position was changed.
   int64 block_height = 7;
+
+  // Exchanged margin is the amount of collateral received by the trader during
+  // the position change. A positve value indicates that the trader received
+  // funds, while a negative value indicates that the trader spent funds.
+  string exchanged_margin = 8 [
```

Comment:
I would name this field `margin_to_user` to make clear what a positive or negative value means.